kratos/python/add_other_utilities_to_python.cpp

```diff
@@ -466,11 +466,12 @@ void AddOtherUtilitiesToPython(pybind11::module &m)
     py::class_<OpenMPUtils >(m,"OpenMPUtils")
         .def(py::init<>())
         .def_static("SetNumThreads", &OpenMPUtils::SetNumThreads)
-    //     .staticmethod("SetNumThreads")
         .def_static("GetNumThreads", &OpenMPUtils::GetNumThreads)
-    //     .staticmethod("GetNumThreads")
         .def_static("PrintOMPInfo", &OpenMPUtils::PrintOMPInfo)
-    //     .staticmethod("PrintOMPInfo")
+        .def_static("GetCurrentNumberOfThreads", &OpenMPUtils::GetCurrentNumberOfThreads)
+        .def_static("GetNumberOfProcessors", &OpenMPUtils::GetNumberOfProcessors)
+        .def_static("IsDynamic", &OpenMPUtils::IsDynamic)
```

Comment:
Also no sense to export this directly from the utils, it will return the default value (false)

Reply:
done